src/api/dashboard-templates.ts

```diff
@@ -103,6 +113,16 @@ export async function getDashboardTemplates(token: string, type?: LayoutTypes):
   return json.data;
 }
 
+export async function getWidgetMapping(token: string): Promise<WidgetMapping> {
+  const resp = await fetch(`/api/chrome-service/v1/dashboard-templates/widget-mapping`, {
+    method: 'GET',
+    headers: getRequestHeaders(token),
```

Comment:
No need to set these. Chrome injects it into all fetch/xhr requests.

Reply:
@Hyperkid123 alright - do you mind if we keep it in this PR and I'll remove in a follow up? I know we override the global FETCH calls.

Reply:
Yep cool